### ISSUE TITLE
Grafisk glitch på forsiden

### ISSUE THREAD

**Issue by tadoh** [ronan-mch/det_sker#324]
When I scroll down on the front page, the column with categories (everything under "Vis Kun") moves a couple of cm to the left after I pass the "Hele kalenderen" line. It jumps back to the right if I scroll up.

I am on FF with windows right now.

**Comment by ronan-mch** [comment]
yeah, I have the same problem. I'd like to get to the bottom of the issue that Sigge reported though in #323 first, as that's somewhat more serious. I take it that you don't encounter that problem?

**Comment by tadoh** [comment]
Nej, det bliver stadig highlighted lidt for tykt, men ingen problemer der

Den 02-03-2016 kl. 22:09 skrev Ronan McHugh:

**Comment by Siggik** [comment]
Jeg får også dette problem på min skole computer. Jeg har ikke rettighederne til at se opløsningen, men den er høj:)
